### DISCUSSION TITLE
Zim 0.75.1 released

### DISCUSSION THREAD

**Discussion by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#2210] [Announcements]
Dear all,

Just released the first "dot" update for the 0.75 release. Specifically a number of bugs with startup are fixes.

Regards,

Jaap

##  0.75.1 - Tue 25 Oct 2022
* Fix bug with relative path notebook icon
* Fix bug with icons in notebookdialog
* Fix bug with loading sub-notebooks via "notebook.zim"
* Fix bug with "--list" commandline
* Fix issue opening unicode attachements on win32
* Fix saving preferences for edit bar
* Fix link visibility in TableEditor
* Use flatpak locale directory in flatpak

**Comment by dehesselle** [comment]
Updated macOS app has been released.

- macOS Ventura isn't supported at this time

**Reply by dehesselle** [reply]
~macOS Ventura is now supported~ Zim does no longer crash on Ventura (version `0.75.1+227` and up). But there is also a serious upstream issue that makes the GUI unresponsive to mouse events. Keyboard navigation works.

**Comment by ZimTom** [comment]
Thanks a lot for this update. Glad to see the file links in a page keep working after deliberately renaming the page title to include an umlaut :-)
